app/pages/sections/supplier/select/ui.test.js

```diff
@@ -185,3 +198,15 @@ test('should anchor to the field when clicking on the error link in errorSummary
     .click(errorSummary.find('li a').nth(0))
     .expect(getLocation()).eql(`${pageUrl}#selectSupplier`);
 });
+
+test('should redirect to /organisation/order-id/supplier when ORDAPI returns order data', async (t) => {
+  nock(orderApiUrl)
+    .get('/api/v1/orders/order-id/sections/supplier')
+    .reply(200, orderData);
+
+  await pageSetup(t, true, orderData);
```

Comment:
change this to ` await pageSetup(t, true, true, orderData);`
to set up the page with the session state